libraries/cyclestreets-view/src/main/java/net/cyclestreets/content/RouteData.java

```diff
@@ -7,16 +7,20 @@
   private final String name;
   private final String json;
   private final Waypoints points;
+  private final Boolean saveRoute;
```

Comment:
Any particular reason to make this a `Boolean` object rather than a primitive `boolean`?  Same goes for the constructor param, obviously

Reply:
No, no reason - it can be a primitive.